docs/source/usage.rst

```diff
@@ -300,9 +302,9 @@ Example:
 Model parameter files
 =====================
 
-|Software| outputs three model parameter files, ``.maf``, ``.seqError``, ``.genoError``. These give the minor allele frequency, sequencing error rates, and genotyping error rates used. All three files contain a single column with an entry for each marker.
+|Software| outputs three model parameter files, ``.alt_allele_prob.txt``, ``.seqError``, ``.genoError``. These give the minor allele frequency, sequencing error rates, and genotyping error rates used. All three files contain a single column with an entry for each marker.
```

Comment:
@AprilYUZhang we should change docs for these error file outputs too, right?

Reply:
Yes, it will be next step. I changed the order a little.

Reply:
@AprilYUZhang is now the time to address these file name changes (`seqErrors` and `genoErrors`)?